test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -26,119 +26,127 @@ $powershellCoreModules = @(
 
 # The file extension for the help content on the Download Center.
 # For Linux we use zip, and on Windows we use $extension.
-$extension = ".zip"
+[string] $extension = ".zip"
 
 if ([System.Management.Automation.Platform]::IsWindows)
 {
     $extension = ".cab"
 }
 
+[string] $userHelpRoot = $null
+
 if([System.Management.Automation.Platform]::IsWindows)
 {
-    $userHelpRoot = Join-Path $HOME "Documents/PowerShell/Help/"
+    # To the reader: the explicit parameter names below are required by a brainless code checker.
```

Comment:
Please remove the unneeded comment.

Reply:
The comment is here so that a future editor will know that they cannot remove the parameter names, which is what anyone would do seeing this absurd syntax.

Reply:
Makes sense to me, thanks for the note.